airflow/providers/openlineage/extractors/manager.py

```diff
@@ -105,14 +106,25 @@ def extract_metadata(self, dagrun, task, complete: bool = False, task_instance=N
                 task_metadata = self.validate_task_metadata(task_metadata)
                 if task_metadata:
                     if (not task_metadata.inputs) and (not task_metadata.outputs):
-                        self.extract_inlets_and_outlets(task_metadata, task.inlets, task.outlets)
-
+                        if (
+                            IS_AIRFLOW_2_10_OR_HIGHER
```

Comment:
nit: we can probably remove `IS_AIRFLOW_2_10_OR_HIGHER` check from this whole func as the ImportError in `self.get_hook_lineage()` will be raised anyway. But if you think it's worth keeping, we can also move it inside of `self.get_hook_lineage()` and simplify this if.
```suggestion
                        if (hook_lineage := self.get_hook_lineage()) is not None:
```

Reply:
agreed with Kacper 👍

Reply:
Fixed.